README.md

```diff
@@ -75,9 +86,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1. A separate styles.css file. [external]
+2. The <head> section of an HTML document. [embedded]
+3. Within the body of the HTML document. [in-line]
 ```
```

Comment:
:+1:

You may have meant this by the ordering, but inline styling has the highest level of specificity (even though external stylesheets are the preferred method of writing CSS).